src/Cli/Utils.cs

```diff
@@ -898,6 +900,33 @@ public static bool WriteJsonContentToFile(string file, string jsonContent)
             return true;
         }
 
+        /// <summary>
+        /// This method will check if DAB_ENVIRONMENT value is set.
+        /// If yes, it will try to merge dab-config.json with dab-config.{DAB_ENVIRONMENT}.json
+        /// </summary>
+        public static string MergeConfigsIfAvailable()
```

Comment:
>MergeConfigsIfAvailable [](http://example.com/codeflow?start=29&length=23)

would `TryMergeConfigsIfAvailable(out mergedFilePath)` convention all us to more clearly instruct the caller whether the merge succeeded or not?